atomicapp/plugin.py

```diff
@@ -164,3 +166,6 @@ def getProvider(self, provider_key):
             if key == provider_key:
                 logger.debug("Found provider %s", provider)
                 return provider
+        # if 'provider_key' matches to None of the supported providers
+        raise NuleculeException("Invalid Provider - '{}', provided in answers.conf (choose from "
+                                "'docker', 'kubernetes', 'openshift', 'marathon')".format(provider_key))
```

Comment:
Also if anything we should be using the `PROVIDERS` constant from `constants.py` to show them options if we choose to do so.